webapp/src/index.tsx

```diff
@@ -10,7 +11,24 @@ import {PluginRegistry} from './types/mattermost-webapp';
 export default class Plugin {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function
     public async initialize(registry: PluginRegistry, store: Store<GlobalState, Action<Record<string, unknown>>>) {
-        // @see https://developers.mattermost.com/extend/plugins/webapp/reference/
+        const state = store.getState();
+        const config = getConfig(state);
+        const siteURL = config.SiteURL;
+
+        const endpoint = `${siteURL}/plugins/${manifest.id}/config`;
+        fetch(endpoint).then((r) => r.json()).then((data) => {
+            // Note that you should store the result in redux for later use.
+            const message = `Plugin received config through API: ${JSON.stringify(data)}`;
+            alert(message);
+        });
+
+        const websocketEventName = 'setting_changed';
+        const fullEventName = `custom_${manifest.id}_${websocketEventName}`;
```

Comment:
Notice that the end of the event name matches what was sent by the plugin's server